### PR TITLE
Fix safari ratio issue + add mobile specific opti

### PR DESCRIPTION
**Pull request by mvaivre**

### CODE REVIEW

**Comment by nop33** [comment]
I cannot test the charts fix locally, cause of CORS policy:

```
Access to fetch at 'https://mainnet-backend.alephium.org/charts/transactions-count?fromTs=1651503006878&toTs=1654095006878&interval-type=daily' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

As for the small mobile screen, I would really just stack them when the width is < 368px, the problem is still there.

**Comment by mvaivre** [comment]
> As for the small mobile screen, I would really just stack them when the width is < 368px, the problem is still there.

I'll do that. I'll need a new breakpoint preset but that's fine.

**Comment by mvaivre** [comment]
@nop33 done

File: src/style/globalStyles.ts [hunk 1]
```diff
@@ -22,16 +22,21 @@ import tinycolor from 'tinycolor2'
...
-}
+export const deviceBreakPoints = Object.entries(deviceSizes).reduce<{ [Key in keyof typeof deviceSizes]?: number }>(
+  (a, s) => ({
+    ...a,
+    [s[0]]: `(max-width: ${s[1]}px)`
+  }),
+  {}
+)
+
+console.log(deviceBreakPoints)
```
Comment (nop33) [inline src/style/globalStyles.ts:39]: :)

Comment (mvaivre) [reply src/style/globalStyles.ts:39]: WUT? I really thought I deleted it, cmon.

**Review by nop33 — APPROVED** [review]

**Pull request merged**